ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml

```diff
@@ -29925,24 +29189,12 @@ periodics:
     org: openshift
     repo: openshift-tests-private
   labels:
-    ci-operator.openshift.io/cloud: aws
-    ci-operator.openshift.io/cloud-cluster-profile: aws-qe
-    ci-operator.openshift.io/variant: multi-nightly
-    ci.openshift.io/generator: prowgen
-    job-release: "4.12"
-    pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.12-multi-nightly-aws-ipi-amd-f60-netobserv
-  reporter_config:
-    slack:
-      channel: '#team-qe-netobserv'
```

Comment:
Looks like slack notification got removed unexpectedly here ?

Reply:
does this part added manually, I just move the profile from multi file  to amd file, then run `make jobs` to update periodics, I don't remove it specially.

Reply:
Yes, they are added manually currently. Rename and then "make jobs" will lost them. So we will have to add it back manually again.
I'll get a PR out today to get them added on demand.

Reply:
Got it, thanks. I will add them manually for this pr

Reply:
the issue has fixed